mod/03_machines/lib/javascript/wagn.js.coffee

```diff
@@ -150,6 +150,9 @@ $(window).ready ->
   $('body').on 'ajax:success', '.slotter', (event, data, c, d) ->
     unless event.slotSuccessful
       $(this).slotSuccess data
+      # should scroll to top after clicking on new page
+      if $(this).hasClass "card-paging-link"
+        $("body").scrollTop 0
```

Comment:
Right, not the page, but of the part of the page with the pager in it.  I was thinking of the process of going through a list, if you are paging down from the bottom of one page, you want the top of the next and vice versa.  If the page is long and doesn't fit on one screenful, what is the desired UX?  A continuous pager that is more like scrolling is more typically what the designers are asking for.